react-charts/area-chart/step4.md

```diff
@@ -1,61 +1,67 @@
-Now we'll add multiple datasets to the area chart...
+In this step you will add tooltips to the area chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It will look like this:
 
 <pre class="file">
-&lt;ChartArea
-  data={[
-    { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 3 }, 
-    { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 4 }, 
-    { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 8 }, 
-    { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 6 }
-  ]} 
-/&gt;
+&lt;Chart
+  height={250}
+  width={600}
+&gt;
 </pre>
 
-3) Replace that section with the following code:
+3) <strong>Add a `containerComponent` property to that section:</strong>

```

Comment:
Adding `containerComponent={}` breaks the example, which is why I did not split this up into multiple steps. IMO, this does not add value as a separate step.

Reply:
I have combined the `containerComponent={}` and the <ChartVoronoiContainer/> in one step so that it still breaks up the steps, but there is no error.

We just got a lot of feedback that users wanted the steps to be broken up more, but I 100% agree that steps that throw errors shouldn't be split up.